service/controller/resource/instance/delete.go

```diff
@@ -2,10 +2,133 @@ package instance
 
 import (
 	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/microerror"
+	capzv1alpha3 "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
+	capzexpv1alpha3 "sigs.k8s.io/cluster-api-provider-azure/exp/api/v1alpha3"
```

Comment:
Not sure if it matters or should be aligned, but I have been using `expcapzv1alpha3` shorthand for this.

Reply:
Yeah, let's align. We can update them later.